channelserver/src/meta.rs

```diff
@@ -127,14 +127,10 @@ fn get_ua(
     None
 }
 
-fn is_trusted_proxy(proxy_list: &[IpNet], host: &str) -> Result<bool, HandlerError> {
+fn is_trusted_proxy(proxy_list: &[IpNet], host: &IpAddr) -> Result<bool, HandlerError> {
```

Comment:
no need for a Result<bool> now, this can just return a bool (this could also be a one liner: `proxy_list.iter().any(|range| range.contains(host))`)